examples/seismic/utils.py

```diff
@@ -192,7 +192,8 @@ def seismic_args(description):
     parser.add_argument("--nbl", default=40,
                         type=int, help="Number of boundary layers around the domain")
     parser.add_argument("-k", dest="kernel", default='OT2',
-                        choices=['OT2', 'OT4', 'centered', 'staggered'],
+                        choices=['OT2', 'OT4', 'centered', 'staggered',
+                                 'blanch_symes', 'ren', 'deng_mcmechan'],
```

Comment:
these ones only apply to viscoacoustic

Reply:
So now, after migrating options to seismic_args we need to find a more 'global' way to do that, how should that be handled?

Reply:
Yes, same way `--fs` or `--noazimut` is done in acoustic and tti, example specific args in each example with according checks.

Reply:
But kernel options needs to be in a dict right? It is not an extra arg.